tests_integration/test_object.py

```diff
@@ -132,3 +135,55 @@ def test_show_drop_image_repository(runner, test_database, snowflake_session):
     )
     assert result_drop.exit_code == 0, result_drop.output
     assert f"{repo_name} successfully dropped" in result_drop.output
+
+
+@pytest.mark.parametrize(
+    "object_type,object_definition",
+    [
+        ("database", {}),
+        (
+            "schema",
+            {
+                "name": "test_schema",
+                "comment": "test_schema = schema for testing stuff",
+            },
+        ),
+        ("image-repository", {"name": "test_image_repo"}),
+    ],
+)
+@pytest.mark.integration
```

Comment:
Add integration tests for expected errors so we have a check that nothing changes on server side